packages/reason-react-ppx/reason-react-ppx.0.12.0/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "React.js JSX PPX"
+description: "ReasonReact JSX PPX"
+maintainer: [
+  "David Sancho <dsnxmoreno@gmail.com>"
+  "Antonio Monteiro <anmonteiro@gmail.com>"
+]
+authors: [
+  "Cheng Lou <chenglou92@gmail.com>" "Ricky Vetter <rickywvetter@gmail.com>"
+]
+license: "MIT"
+homepage: "https://reasonml.github.io/reason-react"
+doc: "https://reasonml.github.io/reason-react"
+bug-reports: "https://github.com/reasonml/reason-react/issues"
+depends: [
+  "dune" {>= "3.9"}
+  "ocaml" {>= "4.13.0"}
+  "reason" {>= "3.6.0" & < "3.10.0"}
+  "ppxlib" {>= "0.28.0"}
+  "merlin" {with-test}
```

Comment:
you probably want to restrict the version of merlin to `4.10-414` for the tests
```suggestion
  "merlin" {= "4.10-414" & with-test}
```

Reply:
Since Melange v1 is attached is unlikely to go for another version, but yes, generally good.